bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -1161,4 +1166,25 @@ spec:
   maturity: stable
   provider:
     name: Red Hat
+  relatedImages:
```

Comment:
would be nice to:
- figure out where this is coming from / why?
- redis and psql have no "name" values - why
- check midstream for possible clash when updating these values for prod release.